.mailmap

```diff
@@ -0,0 +1,27 @@
+Jonathan Schneider <jkschneider@gmail.com> Jon Schneider <jkschneider@gmail.com>
+Jonathan Schneider <jkschneider@gmail.com> Jon Schneider <jschneider@gradle.com>
+Jonathan Schneider <jkschneider@gmail.com> Jon Schneider <jschneider@gradle.com>
+Jonathan Schneider <jkschneider@gmail.com> Jonathan Schnéider <jkschneider@gmail.com>
+Tyler Van Gorder <tkvangorder@users.noreply.github.com> <1878529+tkvangorder@users.noreply.github.com>
+Patrick Way <patway99@gmail.com> Patrick <patway99@gmail.com>
+Patrick Way <patway99@gmail.com> <pway99@users.noreply.github.com>
+Tracey Yoshima <tracey.yoshima@gmail.com>
+Aaron Gershman <aegershman@gmail.com> <5619476+aegershman@users.noreply.github.com>
+Knut Wannheden <knut@moderne.io> <knut.wannheden@gmail.com>
+Knut Wannheden <knut@moderne.io> <knut.wannheden@mobi.ch>
+Kevin McCarpenter <kevin@moderne.io> Kevin McCarpenter™️ <kevin@moderne.io>
+Kun Li <kun@moderne.io> <122563761+kunli2@users.noreply.github.com>
+Jonathan Leitschuh <jonathan.leitschuh@gmail.com> <Jonathan.Leitschuh@gmail.com>
+Jon Brisbin <jon@jbrisbin.com> <jon@moderne.io>
+Scott Jungling <scott@moderne.io> <scott.jungling@gmail.com>
+Aurélien Mino <aurelien.mino@gmail.com> <155828+murdos@users.noreply.github.com>
+Alex Boyko <aboyko@vmware.com> BoykoAlex <aboyko@pivotal.io>
+Alex Boyko <aboyko@vmware.com> <aboyko@pivotal.io>
+Mike Solomon <mike@moderne.io> <mikesol@hey.com>
+Nate Danner <nate@moderne.io> <nate.danner@gmail.com>
+Reagan Karnes <reagan@moderne.io> Reagan K <30000481+reka18@users.noreply.github.com>
+Reagan Karnes <reagan@moderne.io> Reagan K <reagan.karnes@tausight.com>
+Sean C. Sullivan <github@seansullivan.com> sullis <github@seansullivan.com>
+Raquel Pau <raquel@moderne.io> <1483433+rpau@users.noreply.github.com>
+Olga Kundzich <olga@moderne.io> <okundzich@gmail.com>
+Tim te Beek <tim@moderne.io> <tim.te.beek@jdriven.com>
```

Comment:
Would this be the format to also capture my other email address contributions?
```suggestion
Tim te Beek <tim@moderne.io> <tim.te.beek@jdriven.com>
Tim te Beek <tim@moderne.io> <timtebeek@gmail.com> 
```

Reply:
Thanks for the update. There are probably more email addresses missing here. Hopefully we will in a next iteration be able to access GitHub APIs and get the user accounts from there for the individual commits. That would allow us to simply generate the GitHub account name into the resource descriptor and then the UI could display a link to the account page, an avatar or whatever.